src/extensions/spikeamplitudes/SpikeAmplitudesView/SpikeAmplitudesPanel.ts

```diff
@@ -23,7 +23,7 @@ class SpikeAmplitudesPanel {
     _calculationScheduled: boolean = false
     _calculationError: Error | null = null
     _yrange: {min: number, max: number} | null = null
-    _panelGroup: SpikeAmplitudesPanel[] | null = null
+    _globalAmplitudeRange: {min: number, max: number} | null = null
```

Comment:
Changed method for determining the auto-amplitude range for the spike amplitude plot. There was a problem (react call stack overflow) with the old method, and this is more straightforward anyway